new/chef-workstation.md

```diff
@@ -0,0 +1,108 @@
+---
+RFC: unassigned
+Title: The Future of Chef Workstation
+Author:
+- Jon Morrow <jmorrow@chef.io>
+- Noah Kantrowitz <noah@coderanger.net>
+Status: Draft
+Type: Informational
+---
+
+# The Future of Chef Workstation
+
+ChefConf 2018 saw the unveiling of both the `chef-run` command and a Chef Workstation
+installer package containing it. This has lead to user confusion on two fronts.
+First, the term "Chef run" (or "Chef client run") is already in very common use
+in our community, so trying to communicate about `chef-run` has been tricky.
+Second, having two installers (Chef Workstation and ChefDK) has resulted in a
+lack of certainty about which should be used.
+
+## Motivation
+
+    As a Chef user,
+    I want to use the new Chef functionality,
+    so that I can manage machines/resources without Chef Server.
+
+    As a Chef community member,
+    I want to communicate with other Chef community members,
+    so that I can share knowledge and answer questions.
+
+## Specification
+
+1. The `chef-run` command will be renamed to `chef-target` [ed: not attached to this name, suggest a better on in comments].
+2. The `chef-target` gem will be split into its own repository for future development.
```

Comment:
What is the motivation for moving this into its own repository?

Reply:
Because we probably shouldn't have repos called both `chef-dk` and `chef-workstation` if we decided to rename things in one way or the other, and putting it in its own repo will allow for a faster release cycle (if it ends up in the `chef-dk` gem/repo or a renamed echo of that then it would be more bound by the monthly release cycles). Also just in general, doing releases from a monorepo is super awkward for users, you end up with problems like multiple sets of release tags in one repo which can be very non-obvious for users not used to working with git monorepos.